examples/static-hbase-encrypted/kms.tf

```diff
@@ -0,0 +1,82 @@
+locals {
+  effective_kms_key_arn = var.kms_key_arn == "" ? aws_kms_key.kms_encryption_key[0].arn : var.kms_key_arn
+  account_id            = data.aws_caller_identity.current.account_id
+}
+
+data "aws_caller_identity" "current" {}
+
+//The minimal policy document for the KMS key
+data "aws_iam_policy_document" "kms_key" {
+  version = "2012-10-17"
+
+  statement {
+    sid       = "Enable IAM User Permissions"
```

Comment:
For the root account user? I'm not sure I totally understand this statement of the policy.

Reply:
It mentions the account not the root user.
For reference > https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_principal.html

Thank you,